DigitalLearningSolutions.Data.Tests/Services/TutorialContentServiceTests.cs

```diff
@@ -234,6 +234,66 @@ public void Get_tutorial_information_nextSection_skips_sections_full_of_archived
             }
         }
 
+        [Test]
+        public void Get_tutorial_information_nextSection_returns_section_with_only_diagnostic_assessment()
+        {
+            // Given
+            const int candidateId = 74411;
+            const int customisationId = 5852;
+            const int sectionId = 150;
+            const int tutorialId = 634;
+
+            const int nextSectionId = 151; // All tutorials are CustomisationTutorials.Status = 0, though some DiagStatus = 1

```

Comment:
Very minor but I would name this `expectedNextSectionId` 🙂

Reply:
Yep, sorry! Fixed in 08afd86 🙂